src/Template/rector.php

```diff
@@ -122,4 +123,8 @@
     $services->set(MakeInheritedMethodVisibilitySameAsParentRector::class);
     $services->set(SimplifyEmptyArrayCheckRector::class);
     $services->set(NormalizeNamespaceByPSR4ComposerAutoloadRector::class);
+    $services->set(TypedPropertyRector::class)
+        ->configure([
+            TypedPropertyRector::INLINE_PUBLIC => true,
```

Comment:
@samsonasik This should be `false`, shouldn't be?
https://github.com/rectorphp/rector/blob/a7f83696f3c453cfd52f42cf3b9142c1c94be8c1/rules/Php74/Rector/Property/TypedPropertyRector.php#L47

Reply:
We discussed this (I think PMs on Slack?) and the intent was that this Config file be used with projects, where developers can afford to change all their properties without BC concerns. 

Using this in libraries this rule should be changed, or it does create BC issues.

Reply:
@kenjis it depends on what the usage. @MGatner we probably needs a note for it in the readme.

Reply:
I sent a PR #25

Reply:
@MGatner @samsonasik Thanks!